plugins/abci_indexing/sqs/sqs_client.go

```diff
@@ -0,0 +1,102 @@
+package pluginsqs
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"strconv"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/sqs"
+
+	types "github.com/sedaprotocol/seda-chain/plugins/abci_indexing/types"
+)
+
+var queueURLEnvName = "SQS_QUEUE_URL"
+
+type SqsClient struct {
+	queueURL  string
+	sqsClient *sqs.SQS
+}
+
+func (client *SqsClient) sendMessageBatch(batch []*sqs.SendMessageBatchRequestEntry) ([]*sqs.BatchResultErrorEntry, error) {
```

Comment:
`sc` might work better as receiver name according to https://google.github.io/styleguide/go/decisions.html#receiver-names